libs/utils/utils.js

```diff
@@ -772,7 +782,10 @@ function decorateHeader() {
   const dynamicNavActive = getMetadata('dynamic-nav') === 'on'
     && window.sessionStorage.getItem('gnavSource') !== null;
   if (!dynamicNavActive && (baseBreadcrumbs || breadcrumbs || autoBreadcrumbs)) header.classList.add('has-breadcrumbs');
-  if (getMetadata('is-localnav') === 'true' && getMetadata('mobile-gnav-v2') !== 'false') {
+  const gnavSource = await getGnavSource();
+  const gnavName = gnavSource.split('/').pop();
```

Comment:
Do we really need to split? We could just match `\/localnav\/` in the regex on line 788 below

Reply:
I would still prefer using split and pop since it is more readable. For allocation i can change it to 
`gnavSource.split('/').pop().match('localnav')`

Reply:
`split` will allocate an array on the heap, but it should be negligible enough.